src/gui/qgsdialog.cpp

```diff
@@ -34,7 +34,7 @@ QgsDialog::QgsDialog( QWidget *parent, Qt::WindowFlags fl,
   else
     layout = new QHBoxLayout();
   mLayout = new QVBoxLayout();
-  layout->addItem( mLayout );
+  layout->addLayout( mLayout );
```

Comment:
This method doesn't exist on QLayout, only on QBoxLayout, so it won't compile

Reply:
The "layout" variable here seems to be assigned QVBoxLayout or QHBoxLayout, is that wrong?

Reply:
no, but still *layout* is a QLayout.

Did you manage to make it compile ? On what platform?